esx_service/volume_kv.py

```diff
@@ -82,7 +84,7 @@
 ACCESS_TYPES = [ACCESS_READWRITE, ACCESS_READONLY]
 
 # Filesystem type
-# This option is handled in the volume-plugin at the docker host, and tracked in volume metadata. 
+# This option is handled in the volume-plugin at the docker host, and tracked in volume metadata.
```

Comment:
I use Visual Studio Code. Trailing spaces trimming is configurable there, and by default they are not trimmed (which was annoying :-) )